vana/contracts/__init__.py

```diff
@@ -2,5 +2,11 @@
     "satori": {
         "TeePool": "0x88790ffF10E952ffc13Be22a442616eAfE081594",
```

Comment:
Can you please update Satori too, I believe there's new deployments (and new addresses, which match the address in Moksha). 

The ABIs need to be updated too, you can do that by running `poetry run python -m vana.contracts`